### PR TITLE
added table files

### PR DESCRIPTION
**Pull request by AshiraFriedman**
Please review my work and comment in this pull request with any corrections.
Once the tables and data are approved, then I can work on the reports.
Thank you.
Note that this pull request is in the main branch.

### CODE REVIEW

**Review by davidrose777 — COMMENTED** [review]
Hi

**Comment by davidrose777** [comment]
Ok

**Comment by AshiraFriedman** [comment]
That comment above of mine was from a while ago.
If you want, you can comment in this pull request and finish the process and then the grade can be a 90% or you can just move on to the next session and the mark can stay at 80% - whatever you prefer

**Comment by davidrose777** [comment]
Yes I will comment after I submit my next HW BH if that's ok, I'm almost finished with it won't take long. Is that ok? TX 

Sent from Yahoo Mail on Android 
 
  On Tue, Jul 4, 2023 at 1:25 PM, Ashira ***@***.***> wrote:   


That comment above of mine was from a while ago.
If you want, you can comment in this pull request and finish the process and then the grade can be a 90% or you can just move on to the next session and the mark can stay at 80% - whatever you prefer

**Comment by AshiraFriedman** [comment]
That's fine

**Comment by davidrose777** [comment]
Hi, I hope this is the right place to reach you. I added some comments on the Database itself and have no idea how to get them to reach you.

**Comment by AshiraFriedman** [comment]
Yes, I get an email when you comment here.
The way that I see your comments on the files is by you going to the tab "Files Changed", start a review and make the comments.
Please watch the git video on pull requests which explains exactly what to do.  If you don't understand, a mentor can show you what to do.
If the next submission doesn't have those comments for me to see, I will close this HW issue with a mark of 80% and we will not continue further.

File: Database Files/0 database.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+use master
+go 
+drop database if exists GreenBeretOpsDB
+go
+create database GreenBeretOpsDB
+go
+use GreenBeretOpsDB
+go
```
Comment (davidrose777) [inline Database Files/0 database.sql:8]: Great job.

File: Database Files/1 table.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+GO
+
+drop table if exists Mission
+GO
+
+create table dbo.Mission(
+    MissionId int not null identity primary key,
+    FirstName varchar(35) not null constraint Mission_FirstName_cannot_be_blank check(FirstName > ''),
+    LastName varchar(35) not null constraint Mission_LastName_cannot_be_blank check(LastName > ''),
+    YearSoldierBorn int not null constraint Mission_YearSoldierBorn_must_be_between_1776_and_current_date check(YearSoldierBorn between 1776 and getdate()),
```
Comment (davidrose777) [inline Database Files/1 table.sql:11]: GreenBerets started 1952.

File: Database Files/1 table.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                            when Medal = 'Silver' then 2
+                            when Medal = 'Bronze' then 3 
+      end persisted,
+    MissionImportanceNum int not null constraint Mission_MissionImportance_must_be_between_5_and_29 check(MissionImportanceNum between 5 and 29),
+    MissionImportance as case
+                            when MissionImportanceNum between 5 and 10 then 3
+                            when MissionImportanceNum between 11 and 20 then 2
+                            when MissionImportanceNum between 21 and 37 then 1
+     end persisted,
+    MissionImportanceDesc as case
```
Comment (davidrose777) [inline Database Files/1 table.sql:24]: This was meant to be 'Top Secret' and not added to the database.

File: Database Files/2 data.sql [hunk 4]
```diff
@@ -0,0 +1,41 @@
+use GreenBeretOpsDB
```
Comment (davidrose777) [inline Database Files/2 data.sql:1]: Perfect job with the Data.

**Comment by davidrose777** [comment]
Thank you  for doing my Spec, you are a great mentor.

File: Database Files/1 table.sql (hunk 3 above)
Comment (AshiraFriedman) [reply Database Files/1 table.sql:24]: what was meant to be Top Secret?  The MissionImportanceDesc?

**Comment by AshiraFriedman** [comment]
I would like to start on the reports, but I don't see the spec in this repo.
Can you add the spec so that I can see what reports need to be done?

File: Database Files/1 table.sql (hunk 2 above)
Comment (AshiraFriedman) [reply Database Files/1 table.sql:11]: Where would I add a constraint for that?

**Comment by davidrose777** [comment]
I think you got an email, but just in case I sent the reports. 

Sent from Yahoo Mail on Android 
 
  On Sun, Jul 9, 2023 at 5:07 AM, Ashira ***@***.***> wrote:   


@AshiraFriedman commented on this pull request.

In Database Files/1 table.sql:
> @@ -0,0 +1,33 @@
+use GreenBeretOpsDB
+GO
+
+drop table if exists Mission
+GO
+
+create table dbo.Mission(
+    MissionId int not null identity primary key,
+    FirstName varchar(35) not null constraint Mission_FirstName_cannot_be_blank check(FirstName > ''),
+    LastName varchar(35) not null constraint Mission_LastName_cannot_be_blank check(LastName > ''),
+    YearSoldierBorn int not null constraint Mission_YearSoldierBorn_must_be_between_1776_and_current_date check(YearSoldierBorn between 1776 and getdate()),

Where would I add a constraint for that?

**Pull request merged**